components/__design__/horizontal-logos.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import Image from 'next/image';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+export const HorizontalLogos: StoryObj = {};
+
+export default {
+  title: 'Design System/Logos',
+  component: () => {
+    return (
+      <div>
+        <Image
+          src={`/static/images/logos/horizontal-light.svg`}
+          className="bg-black"
```

Comment:
I'm not sure we want this here - it illustrates the contrast but doesn't show the image in its raw form - which is a departure from the rest of the work I've seen so far. Opinions @nodejs/nodejs-website ?

```suggestion
```

Reply:
![image](https://github.com/nodejs/nodejs.org/assets/298435/debc243a-6e3a-4e2f-ac79-3f5843811de1)

https://www.chromatic.com/review?appId=64c7d71358830e9105808652&number=5879&type=linked&view=changes

Reply:
For me add a dark background is cool